test/queries/spanish_01.txt

```diff
@@ -0,0 +1,201 @@
+a      
```

Comment:
This is probably a very vague query to make. Can you replace this with another word ?